components/button/index.js

```diff
@@ -4,4 +4,5 @@ export { default as LargeButton } from './LargeButton';
 export { default as PrimaryButton } from './PrimaryButton';
 export { default as ButtonGroup } from './ButtonGroup';
 export { default as Group } from './Group';
-export { default as Copy } from './Copy';
\ No newline at end of file
+export { default as Copy } from './Copy';
+export { default as LinkButton } from './LinkButton';
```

Comment:
Can you export `LinkButton` component to the main `index.js` file also?

Reply:
I forgot to add it inside the commit ^^